nautobot/cloud/api/serializers.py

```diff
@@ -10,3 +13,14 @@ class CloudAccountSerializer(NautobotModelSerializer):
     class Meta:
         model = CloudAccount
         fields = "__all__"
+
+
+class CloudTypeSerializer(NautobotModelSerializer):
+    content_types = ContentTypeField(
+        queryset=ContentType.objects.all(),
```

Comment:
I suspect we're going to want a feature flag for restricting this to only relevant content types (e.g CloudService/CloudNetwork as a starting point).